sql/sqlserver/server.go

```diff
@@ -333,3 +620,235 @@ func (s *Server) lookupDatabase(name string) (uint32, error) {
 	}
 	return uint32(gr.ValueInt()), nil
 }
+
+func (s *Server) processColumns(desc *structured.TableDescriptor,
+	node parser.Columns) ([]structured.ColumnDescriptor, error) {
+	if node == nil {
+		return desc.Columns, nil
+	}
+
+	cols := make([]structured.ColumnDescriptor, len(node))
+	for i, n := range node {
+		switch nt := n.(type) {
+		case *parser.StarExpr:
+			return s.processColumns(desc, nil)
+		case *parser.NonStarExpr:
+			switch et := nt.Expr.(type) {
+			case *parser.ColName:
+				// TODO(pmattis): If et.Qualifier is not empty, verify it matches the
+				// table name.
+				var err error
+				cols[i], err = findColumnByName(desc, et.Name)
```

Comment:
Uh-oh, I'm not sure how you did the move of this code, but it differs from what was in `driver/conn.go`. If you look on the master branch you'll see that `findColumnBy{ID,Name}` no longer exist. I wonder what other changes you lost. You might want to take Tobias' advice about re-doing this change and performing separate commits for the code movement and the fix ups. 

Or go over this change with a fine-tooth comb looking for changes you missed in the migration. :(